<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Creation_partie_on</class>
 <widget class="QWidget" name="Creation_partie_on">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="creation_salon_widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="button_retour">
        <property name="font">
         <font>
          <family>Merriweather</family>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>310</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Merriweather Black</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Créer un salon</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Merriweather</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom du salon</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer">
        <property name="font">
         <font>
          <family>Merriweather</family>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>618</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <family>Merriweather</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="font">
         <font>
          <family>Merriweather</family>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>618</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="font">
         <font>
          <family>Merriweather</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre de joueurs max</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Merriweather</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre de joueurs</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="nbrJoueurs">
        <property name="font">
         <font>
          <family>Merriweather</family>
         </font>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="font">
         <font>
          <family>Merriweather</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Partie privé</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Merriweather</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="font">
         <font>
          <family>Merriweather</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="button_create">
        <property name="font">
         <font>
          <family>Merriweather</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Créer</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>309</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
